modules/rpm-ostree/README.md

```diff
@@ -4,12 +4,24 @@ The [`rpm-ostree`](https://coreos.github.io/rpm-ostree/) module offers pseudo-de
 
 The module first downloads the repository files from repositories declared under `repos:` into `/etc/yum.repos.d/`. The magic string `%OS_VERSION%` is substituted with the current VERSION_ID (major Fedora version), which can be used, for example, for pulling correct versions of repositories from [Fedora's Copr](https://copr.fedorainfracloud.org/).
 
+You can also have local repos inserted in this file location (make the directory if it doesn't exist):  
+```
+files/rpm-ostree/my-repository.repo
+files/rpm-ostree/my-image/my-2nd-repository.repo
+```
+
```

Comment:
````suggestion
The module first downloads the repository files from URLs declared under `repos:` into `/etc/yum.repos.d/`. The magic string `%OS_VERSION%` is substituted with the current VERSION_ID (major Fedora version), which can be used, for example, for pulling correct versions of repositories from [Fedora's Copr](https://copr.fedorainfracloud.org/).

You can also add repository files directly into your git repository if URLs are not provided. For example:
```yml
repos:
   - my-repository.repo # copies in .repo file from files/rpm-ostree/my-repository.repo to /etc/yum.repos.d/
```
````